src/modules/dashboard/components/ExpenditurePage/ExpenditureSettings/ExpenditureSettings.tsx

```diff
@@ -12,6 +12,7 @@ import {
 import { Colony } from '~data/index';
 import styles from './ExpenditureSettings.css';
 import UserAvatar from '~core/UserAvatar';
+
```

Comment:
Please, organise imports in the way we do through the whole dapp. I guess we missed it in the beginning. This is the principle:
1) external packages (react, react-intl etc)
2) imports from our modules (`core`, `dashboard`, `data` etc)
3) relative imports going down based on their location (the closest is at the bottom)
There needs to be an empty line between each group (between relative imports of different location as well)